includes/class-wc-calypso-bridge-setup.php

```diff
@@ -299,6 +334,7 @@ public function woocommerce_create_pages_callback() {
 				// Update and Release row.
 				$wpdb->query( 'COMMIT' );
 				wp_cache_delete( $this->option_prefix . $operation, 'options' );
+				wp_cache_flush();
```

Comment:
Why do we need this? Does the menu has cache on the output?

Reply:
Removed